File: level_1/문자열 다루기 기본.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int is_numbers(char *s)
{
    int i = 0;
    while (*(s + i))
    {
        if (*(s + i) < '0' || *(s + i) > '9')
            return 0;
        i++;
    }
    return 1;
}

bool solution(const char *s)
{
    if (is_numbers(s) && (strlen(s) == 4 || strlen(s) == 6))
        return true;
    return false;
}

File: level_1/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {
        if (!(s.length() == 6 || s.length() == 4))
            return false;
        for (int i = 0; i < s.length(); i++) {
            if (!(s.charAt(i) < 'A' || s.charAt(i) > 'z'))
                return false;
        }
        return true;
    }
}